runtime.txt

```diff
@@ -1 +1 @@
-python-3.6.4
+python-3.6.3
```

Comment:
@rogeruiz: wdyt about using `python-3.6.x` so that the buildpack picks the latest minor version on stage?

Reply:
great idea. I'll update this. thanks!

Reply:
I didn't know you could do this, thanks for the heads up @jmcarp!